docs/azure/sdk/includes/implement-user-assigned-identity.md

```diff
@@ -1,10 +1,13 @@
 ---
 ms.topic: include
-ms.date: 08/15/2024
+ms.date: 02/12/2025
 ---
-[DefaultAzureCredential](../authentication/credential-chains.md#defaultazurecredential-overview) is an opinionated, ordered sequence of mechanisms for authenticating to Microsoft Entra ID. Each authentication mechanism is a class derived from the [TokenCredential](/dotnet/api/azure.core.tokencredential?view=azure-dotnet&preserve-view=true) class and is known as a *credential*. At runtime, `DefaultAzureCredential` attempts to authenticate using the first credential. If that credential fails to acquire an access token, the next credential in the sequence is attempted, and so on, until an access token is successfully obtained. In this way, your app can use different credentials in different environments without writing environment-specific code.
 
-To use `DefaultAzureCredential`, add the [Azure.Identity](/dotnet/api/azure.identity) and optionally the [Microsoft.Extensions.Azure](/dotnet/api/microsoft.extensions.azure) packages to your application:
+[!INCLUDE [implement-managed-identity-concepts](implement-managed-identity-concepts.md)]
+
+### Implement the code
+
+Add the [Azure.Identity](/dotnet/api/azure.identity) and optionally the [Microsoft.Extensions.Azure](/dotnet/api/microsoft.extensions.azure) packages to your application:
```

Comment:
This is the same suggestion I provided for the system-assigned doc. But I wonder if it would make more sense to introduce a zone pivot with options: ASP.NET Core and Other.
```suggestion
Add the [Azure.Identity](/dotnet/api/azure.identity) package. In an ASP.NET Core project, also install the [Microsoft.Extensions.Azure](/dotnet/api/microsoft.extensions.azure) package:
```